markdown_lab/core/config.py

```
@@ -0,0 +1,200 @@
+"""
+
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict
+
+
+@dataclass
+class MarkdownLabConfig:
+    """Centralized configuration for all markdown_lab operations.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)